mantidproject/mantid
# Estimates initial parameters for exponential function in Iqtfit

**Description of work.**
Estimate initial parameters for the exponential function(s) in Iqtfit as was implemented already for the stretched exponential. 

**To test:**

1. Load data (e.g. script below)

from mantid.simpleapi import *

Load(Filename='irs26174_graphite002_red.nxs', OutputWorkspace='irs26174_graphite002_red')
Load(Filename='irs26173_graphite002_res.nxs', OutputWorkspace='irs26173_graphite002_res') 
TransformToIqt(SampleWorkspace='irs26174_graphite002_red', ResolutionWorkspace='irs26173_graphite002_res', 
    ParameterWorkspace='irs26174_graphite002_TransformToIqtParameters', OutputWorkspace='irs26174_graphite002_iqt')

2. Open the Indirect >Data Analysis interface
3. Go to the I(Q,t) Fit
4. Load sample from workspace (select only valid workspace)
5. Add 1 and 2 exponential functions and check that the initial parameters are not (Height=1, Lifetime=1) and are different for a few spectra. The height of the second exp should be 10% of the first and the lifetime should be double (arbitrary choice of second exp initial parameters)

![image](https://user-images.githubusercontent.com/55979119/68217017-c9caad80-ffd9-11e9-9602-372dfaff77c9.png)


Fixes #27200 

This does not require release notes because it is a small change.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
